pyxem/tests/test_signals/test_diffraction2d.py

```diff
@@ -154,9 +114,7 @@ def test_azimuthal_integral_4D(self, test_dp4D):
 
     @pytest.fixture
     def axes_test_dp(self):
```

Comment:
can we remove the word 'test' from this name, perhaps `axes_carry_through_dp` or similar